### ISSUE TITLE
Can not run on iOS after upgrading to Flutter 1.20.3

### ISSUE THREAD

**Issue by idrats** [iqbalmineraltown/flutter_sms_receiver#7]
flutter run output:

```
Failed to build iOS app
Error output from Xcode build:
↳
    2020-09-10 18:22:26.600 xcodebuild[84348:7889314]  DTDeviceKit: deviceType from 00008020-001270961104003A was NULL
    2020-09-10 18:22:26.746 xcodebuild[84348:7889314]  DTDeviceKit: deviceType from 00008020-001270961104003A was NULL
    ** BUILD FAILED **


Xcode's output:
↳
    Command CompileSwift failed with a nonzero exit code
    /Users/idrats/.pub-cache/hosted/pub.dartlang.org/sms_receiver-0.3.1+1/ios/Classes/SmsReceiverPlugin.m:6:4: error: unknown receiver 'SwiftSmsReceiverPlugin'; did you mean 'SmsReceiverPlugin'?
      [SwiftSmsReceiverPlugin registerWithRegistrar:registrar];
       ^~~~~~~~~~~~~~~~~~~~~~
       SmsReceiverPlugin
    In file included from /Users/idrats/.pub-cache/hosted/pub.dartlang.org/sms_receiver-0.3.1+1/ios/Classes/SmsReceiverPlugin.m:1:
    /Users/idrats/.pub-cache/hosted/pub.dartlang.org/sms_receiver-0.3.1+1/ios/Classes/SmsReceiverPlugin.h:3:12: note: 'SmsReceiverPlugin' declared here
    @interface SmsReceiverPlugin : NSObject<FlutterPlugin>
               ^
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
```

flutter doctor: 
```[✓] Flutter (Channel stable, 1.20.3, on Mac OS X 10.15.5 19F101, locale ru-RU)
    • Flutter version 1.20.3 at /Users/idrats/Documents/flutter
    • Framework revision 216dee60c0 (9 days ago), 2020-09-01 12:24:47 -0700
    • Engine revision d1bc06f032
    • Dart version 2.9.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/idrats/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.3

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Connected device (1 available)
    • iPhone (Ilya) (mobile) • 00008020-001270961104003A • ios • iOS 13.5.1

• No issues found!
```

**Comment by iqbalmineraltown** [comment]
Hi, thanks for creating PR for this. Will try to check and update ASAP.

FYI initially I would like to cover iOS counterpart for autofill OTP received.

But after seeing https://github.com/flutter/flutter/issues/13015#issuecomment-624921951, seems like this plugin will be no longer needed as from flutter itself will support this feature. 

This plugin might be deprecated in the near future.

**Comment by Janaka-Steph** [comment]
AutoFillGroup is cool but still buggy though https://github.com/flutter/flutter/issues/65144
